voctogui/lib/audioleveldisplay.py

```diff
@@ -129,10 +141,13 @@ def normalize_db(self, db):
     def clamp(self, value, min_value=0, max_value=1):
         return max(min(value, max_value), min_value)
 
-    def level_callback(self, rms, peak, decay):
-        if self.levelrms != rms or self.levelpeak != peak \
-                or self.leveldecay != decay:
-            self.levelrms = rms
-            self.levelpeak = peak
-            self.leveldecay = decay
+    def level_callback(self, rms, peak, decay, stream):
+        p = self.channels * stream
```

Comment:
`p` is a non descriptive name. Maybe `meter_offset` or `first_channel_index` would be more explanatory.